pkgs/by-name/ti/tiddit/package.nix

```
@@ -0,0 +1,75 @@
+{
+  lib,
+  stdenv,
+  bwa,
+  zlib,
+  python3,
+  fetchFromGitHub,
+  python3Packages,
+}:
+let
+  fermi2 = stdenv.mkDerivation {
+    pname = "fermi2";
+    version = "unstable-20210521";
```

```suggestion
    version = "0-unstable-2021-05-21";
```

see https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming